### ISSUE TITLE
Create a landing page that easily links users to specific Guidance pages

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#1544]
As a user, I need to be able to do find the specific page of the Guidance that I need, so that I can learn how to address the specific issue that I'm working on.

WHY are we building?
Currently there's no way to quickly access a specific page within the Guidance.

WHAT are we building?
A landing page that takes users to the specific page of the Guidance that they want to read.

VALUE created by our solution
Users will more easily find what they need at the time that they need it. 

## Documentation and Artifacts
[Anik's does for Forms](https://articles.alpha.canada.ca/forms-formulaires/guidance/)

## Acceptance Criteria
Given users need to easily find guidance when they need it, when they navigate to a Guidance landing page, there's a quick way to review and select a particular Guidance page

* A11y
* Bilingualism
* Privacy considerations
* Security controls in place
* Measuring success and metrics

## Related [Research Airtable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwQUX3tUxbMxYS7C?blocks=hide) records
- (Ping Adrianne to add stuff here or find stuff yourself)

## QA Steps
- [x] Tested in a realistic production scenario

## GC Articles Publish checklist
- [x]  Supporting feature is released and unflagged
- [x]  Create or update slug
- [x]  Page content and slug is translated
- [x]  Document links to and from the page
- [x]  Add page slug and title to [inventory](https://docs.google.com/document/d/1N6B4fSwoM40m-pDLI70BvLfL7O0n04ndyAXYaIwNbNw/edit)
- [x]  If new page, add WAF rule
- [x]  Create new page or update existing page content
- [x]  Create or update translation to the page
- [x]  Link translations by choosing an assigned translation
- [x]  If new page, save as draft
- [x]  If existing page, update the page
- [x]  Post page in Slack for last minute reviews
- [x]  On staging: Clear the cache through GC Notify's admin panel
- [x]  On staging: Check links, images and language toggle
- [x]  Give a heads up to the app-team channel once ready to publish
- [x]  On staging and production: Clear the cache through GC Notify's admin panel
- [x]  Clear Cloudfront cache in GCA as needed (Andrew and Phil are GCA admins)
- [x]  Update sitemap with new links
- [x]  Update A11y statement by removing issue of multiple ways (Only 1 way to navigate to different pages)

**Comment by YedidaZalik** [comment]
Phil and Yedida met about  site map yesterday
We have not worked on this card yet
Site map will help us do this work

**Comment by jzbahrai** [comment]
Likelihoods are we are not going to implement this as we will have a site map

**Comment by YedidaZalik** [comment]
This morning Phil and Yael told me we probably will need this

I did a first draft of guidance landing page on second page of [this document](https://docs.google.com/document/d/1Q9M2wAgchvfXIn7wLdrLbWnJ1n5q3Jf76_xcKkUk3mQ/edit), after Anik's

Note that we probably want to link to both this page and the Ally statement from the top of the Contact us page, where we link to other documents that answer questions

**Comment by YedidaZalik** [comment]
Anik created the Forms landing page in GC Articles https://articles.alpha.canada.ca/forms-formulaires/guidance/

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/1544?planning-poker) with Zenhub @amazingphilippe @YedidaZalik

**Comment by YedidaZalik** [comment]
@amazingphilippe @yaelberger-commits I ought to have tagged you to take a look at page 2 of [this document](https://docs.google.com/document/d/1Q9M2wAgchvfXIn7wLdrLbWnJ1n5q3Jf76_xcKkUk3mQ/edit#heading=h.lcktk2xfyy26)

**Comment by YedidaZalik** [comment]
Reviewed with Phil and MSB on Monday

**Comment by yaelberger-commits** [comment]
LGTM!

**Comment by yaelberger-commits** [comment]
To be translated after A11y statement

**Comment by YedidaZalik** [comment]
Sent document to MSB for when she's completed the A11y statement

[Bilingual guidance landing page](https://docs.google.com/document/d/1toUmd02vv32h3wipRxBSZMTLYdk-ik171Q0_sCJPWtM/edit#heading=h.xlhsmvfkm272)

**Comment by YedidaZalik** [comment]
Anik gave access to the parallel page for Forms
Copied it and created [reference] Guidance-draft in GC Articles in EN
Andrew gave us access to view it here
https://qrxlqsdhhf5ak44jfbnxkjsshu0yxpbl.lambda-url.ca-central-1.on.aws/reference-guidance

Met with Phil to fix focus states, rounded corners, spacing could be tighter and columns are not equal

Phil will keep working on spacing for part in blue at the end

Since it's a completely new page, once both EN and FR are ready, will need to ask Andrew to fix firewall rules

**Comment by yaelberger-commits** [comment]
Waiting on translation, will publish once ready either today or next week

**Comment by YedidaZalik** [comment]
Started FR page with the content we already have (eg. names of the various guidance pages); need to add in missing content when it's ready and rename pages

**Comment by andrewleith** [comment]
- Fr has come back, will be put in GCA today

**Comment by YedidaZalik** [comment]
Updated GCA with French content

Next need WAF rules

The pages currently titled Guidance/Guides de référence will be renamed Getting started/Découvrir Notify GC with new urls http://notification.canada.ca/getting-started and http://notification.canada.ca/decouvrir-notification-gc

And new EN and FR landing pages with the blocks will take over the titles Guidance/Guides de référence and also take over the urls https://notification.canada.ca/guidance and https://notification.canada.ca/guides-reference

The new landing pages are currently in GC Articles as [reference] Guidance-draft -EN with url https://articles.alpha.canada.ca/notification-gc-notify/reference-guidance/ and
 [reference] Guides de référence -draft-FR with url https://articles.alpha.canada.ca/notification-gc-notify/reference-guides-de-reference-draft-fr/

**Comment by andrewleith** [comment]
- PR for WAF rules will go out this morning

**Comment by YedidaZalik** [comment]
A11y statement is updated in both languages
